src/handlers/swap_handlers.ts

```diff
@@ -30,7 +30,51 @@ export class SwapHandlers {
         this._swapService = swapService;
     }
     public async getSwapQuoteAsync(req: express.Request, res: express.Response): Promise<void> {
-        // parse query params
+        res.status(HttpStatus.OK).send(await this._calculateSwapQuoteAsync(parseGetSwapQuoteRequestParams(req)));
+    }
+    // tslint:disable-next-line:prefer-function-over-method
+    public async getSwapTokensAsync(_req: express.Request, res: express.Response): Promise<void> {
+        const tokens = TokenMetadatasForChains.map(tm => ({
+            symbol: tm.symbol,
+            address: tm.tokenAddresses[CHAIN_ID],
+            name: tm.name,
+            decimals: tm.decimals,
+        }));
+        const filteredTokens = tokens.filter(t => t.address !== NULL_ADDRESS);
+        res.status(HttpStatus.OK).send({ records: filteredTokens });
+    }
+    // tslint:disable-next-line:prefer-function-over-method
+    public async getSwapPriceAsync(req: express.Request, res: express.Response): Promise<void> {
+        const params = parseGetSwapQuoteRequestParams(req);
+        if (params.rfqt === undefined) {
+            params.rfqt = {
+                intentOnFilling: false,
+            };
+        }
+        params.skipValidation = true;
+        const quote = await this._calculateSwapQuoteAsync(params);
+        const { price, value, gasPrice, gas, protocolFee, buyAmount, sellAmount, sources, orders } = quote;
+        logger.info(`Serving indicative quote based on the following orders: ${JSON.stringify(orders)}`);
```

Comment:
This would get a little verbose. Can we remove it?

Reply:
Addressed in https://github.com/0xProject/0x-api/pull/172/commits/b2f6fe76188ca9bfff737cb40a56826366554f5f